Lib/test/cov.py

```diff
@@ -0,0 +1,22 @@
+"""A minimal hook for gathering line coverage of the standard library."""
```

Comment:
Please explain that it's register itself as soon as it's imported and is designed to be used with ``-X presite=``.

You may also list limitations like: filenames are not normalized.